modules/op-specifying-pvc-as-volumesource-in-workspaces.adoc

```diff
@@ -0,0 +1,41 @@
+// This module is included in the following assembly:
+//
+//  *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
+
+[id="specifying-pvc-as-volumesource-in-workspaces_{context}"]
+= Specifying PersistentVolumeClaims as VolumeSource in Workspaces
+
+Workspaces help share data among Tasks and allow you to specify one or more volumes that each Task in the Pipeline requires during execution.
+
+In this section, you will create a PersistentVolumeClaim to provide data storage and bind it to the workspace. This PersistentVolumeClaim provides the volumes or filesystem required for the pipeline execution.
+
+[discrete]
+.Procedure
+
+. Copy the contents of the following sample PersistentVolumeClaim YAML file and save it:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: source-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+----
++
+. Create the PersistentVolumeClaim:
++
+----
+$ oc create -f <PersistentVolumeClaim-yaml-file-name.yaml>
+----
++
+Alternatively, you can also execute the YAML file directly from the Git repository:
++
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml
```

Comment:
This needs to be updated to tech-preview branch... https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-tech-preview-2/01_pipeline/03_persistent_volume_claim.yaml